File: Kattis/leftbeehind.java
// https://open.kattis.com/problems/leftbeehind

import java.util.Scanner;

public class leftbeehind {

    public static void main(String[] args) {

        Scanner scan = new Scanner(System.in);

        while (true) {
            int firstNum  = scan.nextInt();
            int secondNum = scan.nextInt();
            if ((firstNum == 0) && (secondNum == 0)) {
                break;
            }
            if (firstNum + secondNum == 13) {
                System.out.println("Never speak again.");
            } else if (firstNum > secondNum) {
                System.out.println("To the convention.");
            } else if (firstNum == secondNum) {
                System.out.println("Undecided.");
            } else {
                System.out.println("Left beehind.");
            }
        }
    }
}

File: Kattis/leftbeehind.cpp
// https://open.kattis.com/problems/leftbeehind
//
#include <bits/extc++.h>

using namespace std;
using namespace __gnu_pbds;

int main() {

    int x, y;
    while (~scanf("%d %d", &x, &y)) {
        if (x == 0 && y == 0) {
            break;
        }
        if (x + y == 13) {
            printf("Never speak again.\n");
        } else if (x > y) {
            printf("To the convention.\n");
        } else if (x == y) {
            printf("Undecided.\n");
        } else {
            printf("Left beehind.\n");
        }
    }

    return 0;
}
